planning/obstacle_velocity_limiter/README.md

```diff
@@ -0,0 +1,204 @@
+# Obstacle Velocity Limiter
+
+## Purpose
+
+This node reduces the velocity of a trajectory around obstacles in order to convey a better feeling of apparent safety to the passengers.
```

Comment:
@maxime-clem The description is not straightforward to me. It sounds like, the vehicle slows down in every situation when driving close to an obstacle. But it only takes effect under certain conditions (in a curve for example). Could you update the description with a nice explanation? Or adding a simple (but easy to understand) figure to show the function of this module in the "purpose" section would be nice. 
(Since the attached figures in the current readme focus on explaining the algorithm, they are not suitable for explaining the purpose of this module)

Reply:
I have updated the description of the purpose and added images that should make it easy to understand what this node does.

Reply:
Thanks!